dockerfiles/Dockerfile-rosetta

```diff
@@ -248,6 +248,9 @@ RUN git pull \
    && go build -o ${HOME}/app/libp2p_helper \
    && go clean --cache --modcache --testcache -r
 
+# HACK: build without special cpu features to allow more people to run coda
+RUN sed -i 's/+bmi2,+adx/-bmi2,-adx/g' src/lib/zexe/snarky-bn382/dune
```

Comment:
This probably wants to patch `src/lib/marlin_plonk_bindings/stubs/dune` as well.